charts/logzio-logs-collector/README.md

```diff
@@ -0,0 +1,134 @@
+# logzio-logs-collector
+
+**In development**
+
+kubernetes logs collection agent for logz.io based on opentelemetry collector
+
+## Prerequisites
+
+- Kubernetes 1.24+
+- Helm 3.9+
+
+Below is the extended README.md including the full configuration table based on the provided `values.yaml`.
+
+* * *
+
+Logz.io Logs Collector for Kubernetes
+=====================================
+
+The `logzio-logs-collector` Helm chart deploys a Kubernetes logs collection agent designed to forward logs from your Kubernetes clusters to Logz.io. This solution leverages the OpenTelemetry Collector, offering a robust and flexible way to manage log data, ensuring that your logging infrastructure scales with your application needs.
+
+Features
+--------
+
+*   **Easy Integration with Logz.io**: Pre-configured to send logs to Logz.io, simplifying setup and integration.
+*   **Secure Secret Management**: Option to automatically manage secrets for seamless and secure authentication with Logz.io.
+
+Getting Started
+---------------
+
+### Add Logz.io Helm Repository
+
+Before installing the chart, add the Logz.io Helm repository:
+
+```
+helm repo add logzio https://helm.logz.io helm repo update
```

Comment:
Current comands doesn't work for me, consider changing to our existing solutions commands. 
```
helm repo add logzio-helm https://logzio.github.io/logzio-helm
helm repo update

```